rules_cc_windows.patch

```diff
@@ -0,0 +1,524 @@
+diff --git a/cc/private/toolchain/BUILD.tpl b/cc/private/toolchain/BUILD.tpl
```

Comment:
Nit: Even though there is only one patch now that the others have been upstreamed or you've done workarounds, I'd prefer this in a directory, for instance `shared/bazel/patches` or something like that, to make it obvious this is for bazel, and would group all of them together. If we got into a situation where we had multiple patches for a rule we could subdir it again

Reply:
Good call, done!